content/building-blocks/prepare-your-data-for-analysis/data-preparation/large-datasets-R.md

```diff
@@ -2,21 +2,25 @@
 title: "Import Large Datasets Into R"
 description: "Importing large datasets in R is made easy by data.table R package. The package makes importing and manipulating huge datasets (like big data) in R faster"
 weight: 2
-keywords: "import, data, data preparation, big data, large datsets, memory, RAM, data.table, big data, R"
+keywords: "import, data, data preparation, big data, large datsets, memory, RAM, data.table, big data, R, dataframe, object size"
 date: 2021-01-06T22:01:14+05:30
 draft: false
 aliases:
   - /import/large-datsets
 ---
 
 ## Overview
-
-Many R-users rely on the `dplyr` or `read.table` packages to import their datasets as a dataframe. Although this works well for relatively small datasets, we recommend using the `data.table` R package instead because it is significantly faster. This building block provides you with some practical tips for dealing with large datsets in R.
+Many R-users rely on the `dplyr` or `read.table` packages to import their datasets as a dataframe. Although this works well for relatively small datasets, we recommend using the `data.table` R package instead because it is significantly faster. This building block provides you with some practical tips for dealing with large datasets in R.
 
 ## Code
-As a starting point, make sure to clean your working environment in RStudio. Oftentimes, there are datasets stored memory that you have worked with earlier but you're no longer using. Click on the broom icon in the top right window to remove all objects from the environment. 
 
-In addition, switching from the `read.csv()` function to `fread()` can greatly improve the performance of your programme in our experience. Below we illustrate how you can import a (subset of the) data, determine the object size, and store the derivative version of the file for future use.
+Switching from the `read.csv()` function to `fread()` can greatly improve the performance of your program. The code block below illustrates how you can import your whole dataset as well as subsets of it, determine the size of the resulting object, and store your new/clean versions of the data as a new file for future use.
+
+{{% tip %}}
```

Comment:
I would maybe add the tip box after the code chunk in the last?

Reply:
Sounds good! I've moved the tip box to the bottom 👍